services/apps/alcs/src/alcs/search/application/application-search.entity.ts

```diff
@@ -0,0 +1,104 @@
+import {
```

Comment:
suggestion:Rename file to application-search-view.entity.ts so we remembner its a view